StationFinderListenLive.cpp

```diff
@@ -216,7 +220,7 @@ StationFinderListenLive::ParseCountryReturn(BMallocIO* data, const char* searchF
 		doc[matches[5].rm_eo] = 0;
 		doc[matches[6].rm_eo] = 0;
 		doc[matches[8].rm_eo] = 0;
-		printf("Station %s at %s in %s address %s, data rate=%s, genre=%s\n", 
+		printf(B_TRANSLATE("Station %s at %s in %s address %s, data rate=%s, genre=%s\n"), 
```

Comment:
Normally, debug output isn't translated. If you do want to, you may want to use B_TRANSLATE_COMMENT to explain the variables. At least the first 4...
Maybe with an example. I'm guessing: "Station 'station name' at 'URL' in 'country'... hmm, what's the address?

Reply:
I'll check this. I never took a look at this service, because it wasn't even working in first place. It might have moved to a different website or maybe it just went down. Anyway, I'll leave the debug output in English and I'll take a closer look at "ListenLive" to see what happened.